noggin/controller/authentication.py

```diff
@@ -53,3 +56,48 @@ def login():
                     'Could not log in to the IPA server.'
                 ]
     return render_template('login.html', login_form=form)
+
+
+@app.route('/otp/sync/', methods=['GET', 'POST'])
+def otp_sync():
+    form = SyncTokenForm()
+    if form.validate_on_submit():
+        data = {
+            'user': form.username.data,
+            'password': form.password.data,
+            'first_code': form.first_code.data,
+            'second_code': form.second_code.data,
+            'token': form.token.data,
+        }
+
+        url = (
+            'https://'
+            + random.choice(app.config['FREEIPA_SERVERS'])
+            + '/ipa/session/sync_token'
+        )
```

Comment:
do we need to reimplement this code here? can me not use `ipa_admin` or (most likely)  `untouched_ipa_client` here instead, like we do on the register page?

This might also remove the need for using requests here, no?